src/System.CommandLine/Argument{T}.cs

```diff
@@ -122,6 +126,8 @@ public Argument(ParseArgument<T> parse, bool isDefault = false) : this(null, par
         {
         }
 
+        internal override bool HasCustomParser => _hasCustomParser;
```

Comment:
Any reason this is not simply just hard coded to return true?
```suggestion
        internal override bool HasCustomParser => true;
```

Reply:
It only gets set to `true` in the constructors that accept a `ParseArgument<T>`.

Reply:
Ah missed that.